doc/Language/functions.rakudoc

```diff
@@ -1018,6 +1018,55 @@ dispatch, or it could be the inner routine from a C<wrap>.
 Fortunately, we have a series of re-dispatching tools that help us to make
 it easy.
 
+=head2 The Next Tools
+
+The table below represents the next tools in relation to three attributes:
+
+=item1 B<Returns:> "Yes" means that it returns, and the code will continue from
+that point.  "No" means that the it never returns to this function -- it's as 
+though the call has a built-in "return".  
+
+=item1 B<Parameters:> Specifically, whether the parameters from the current
+function are reused in the call, or whether you need to specify some 
+parameters.  In the case of "nextcallee", there are no parameters needed, 
```

Comment:
`C<nextcallee>`

Reply:
Done

Reply:
@raiph I've just done another PR that should address (ie. implement) your comment as well.